### PR TITLE
Update README and add Contribution guide

### PR DESCRIPTION
**Pull request by outsideris**
- Update README
- Add contribution guide
- add a CLA badge

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Thank you for your interest in contributing. As we aim to make it as easy as possible to contribute changes, his guide will introduce basic guidelines on code, documentation and tests that contributors must follow to ensure contributions are accepted quickly.
+
+## Pull Request
+
+The changes should be submitted through 'Pull Request'. Don't send your patches through other channels like email or slack. The submitted PR will be reviewed by the other developers and maintainers and it will be merged by 'Squash Merge'.
+
+We introduce several simple rules for your PR.
+
+* Follow the [7 rules of a great commit message](https://chris.beams.io/posts/git-commit/)
```
Comment (Geod24) [inline CONTRIBUTING.md]: One of those rules is to capitalize your commit message ;)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Contribution Guide For Developers
+
+## How To Contribute
+
+Thank you for your interest in contributing. As we aim to make it as easy as possible to contribute changes, his guide will introduce basic guidelines on code, documentation and tests that contributors must follow to ensure contributions are accepted quickly.
```
Comment (Geod24) [inline CONTRIBUTING.md]: typo: `contribute changes, his guide will...` => should be `this guide`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+The changes should be submitted through 'Pull Request'. Don't send your patches through other channels like email or slack. The submitted PR will be reviewed by the other developers and maintainers and it will be merged by 'Squash Merge'.
+
+We introduce several simple rules for your PR.
+
+* Follow the [7 rules of a great commit message](https://chris.beams.io/posts/git-commit/)
+* When fixing bugs, add tests
+* When providing performance fixes, describe your benchmarking approach so that we can reproduce it
+* Don't include user or tool specific config files or folder, such as .vimrc, .vscode .idea.
+* Any pull request submitted must build and pass tests before being merged.
```
Comment (Geod24) [inline CONTRIBUTING.md]: Trailing dot (`.`) on this line and the previous line, which are inconsistent with the previous bullet points

File: CONTRIBUTING.md (hunk 1 above)
Comment (spikeekips) [reply CONTRIBUTING.md]: ㅋㅋㅋㅋ @outsideris You break the new rules 1st. :)

File: README.md [hunk 4]
```diff
@@ -1,132 +1,14 @@
...
-
-Then you can spawn 3 nodes using the following commands:
-```sh
-docker run --net host --rm -it --env-file=docker/node1.env sebak
-docker run --net host --rm -it --env-file=docker/node2.env sebak
-docker run --net host --rm -it --env-file=docker/node3.env sebak
-```
-
-Changing the number of test nodes should be fairly simple once #46 is solved.
+We welcome contributions by the community. Please open issues for any bug you find, or for enhancement requests. Pull requests are also very welcome. Before submitting, please read the [contribution guidelines](https://github.com/bosnet/sebak/blob/master/CONTRIBUTING.md).
```
Comment (Geod24) [inline README.md]: You can replace the link with just: `[contribution guidelines](CONTRIBUTING.md)`
https://blog.github.com/2013-01-31-relative-links-in-markup-files/

File: CONTRIBUTING.md (hunk 1 above)
Comment (outsideris) [reply CONTRIBUTING.md]: Haha. I got it.

**Comment by outsideris** [comment]
I updated them.

**Pull request merged**